### ISSUE TITLE
After Upgrade to 5.11 on Ubuntu, Suspend is not working anymore

### ISSUE THREAD

**Issue by eddykaya** [jrandiny/yoga-slim7-ubuntu#26]
Hi, 
I recently upgraded to Kernel 5.11, however, the proposed fix from this repository to enable S3 is not working anymore.
What logs should I provide to dig deeper into it?

Cheers,
Eddy

**Comment by eddykaya** [comment]
I reproceed the steps to create acpi_s3_override and grub config, however, system is not usable after wake up. I only got some pixel salad, afterwards, I got a black screen and I can see and use the mouse but nothing more.

Here is my syslog, starting with suspend activation until hard resetting the system:

`Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 31 with keysym 31 (keycode a).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 32 with keysym 32 (keycode b).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 33 with keysym 33 (keycode c).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 34 with keysym 34 (keycode d).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 35 with keysym 35 (keycode e).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 36 with keysym 36 (keycode f).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 39 with keysym 39 (keycode 12).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 38 with keysym 38 (keycode 11).
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: Window manager warning: Overwriting existing binding of keysym 37 with keysym 37 (keycode 10).
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.1355] manager: sleep: sleep requested (sleeping: no  enabled: yes)
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.1359] device (p2p-dev-wlp1s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Feb 26 23:35:01 eddys-pc ModemManager[1119]: <info>  [sleep-monitor] system is about to suspend
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.1368] manager: NetworkManager state is now ASLEEP
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.1371] device (wlp1s0): state change: activated -> deactivating (reason 'sleeping', sys-iface-state: 'managed')
Feb 26 23:35:01 eddys-pc whoopsie[1768]: [23:35:01] offline
Feb 26 23:35:01 eddys-pc dbus-daemon[1010]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.13' (uid=0 pid=1011 comm="/usr/sbin/NetworkManager --no-daemon ")
Feb 26 23:35:01 eddys-pc systemd[1]: Starting Network Manager Script Dispatcher Service...
Feb 26 23:35:01 eddys-pc dbus-daemon[1010]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Feb 26 23:35:01 eddys-pc systemd[1]: Started Network Manager Script Dispatcher Service.
Feb 26 23:35:01 eddys-pc kernel: [  241.744358] wlp1s0: deauthenticating from dc:39:6f:f9:f8:e1 by local choice (Reason: 3=DEAUTH_LEAVING)
Feb 26 23:35:01 eddys-pc wpa_supplicant[1055]: wlp1s0: CTRL-EVENT-DISCONNECTED bssid=dc:39:6f:f9:f8:e1 reason=3 locally_generated=1
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.1664] device (wlp1s0): state change: deactivating -> disconnected (reason 'sleeping', sys-iface-state: 'managed')
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Withdrawing address record for 2003:e9:2714:2000:4143:2297:ff8a:422d on wlp1s0.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Withdrawing address record for 2003:e9:2714:2000:f4e7:cc38:c973:8063 on wlp1s0.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Leaving mDNS multicast group on interface wlp1s0.IPv6 with address 2003:e9:2714:2000:f4e7:cc38:c973:8063.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Joining mDNS multicast group on interface wlp1s0.IPv6 with address fe80::3e3c:bf3d:da61:7ea6.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Registering new address record for fe80::3e3c:bf3d:da61:7ea6 on wlp1s0.*.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Withdrawing address record for fe80::3e3c:bf3d:da61:7ea6 on wlp1s0.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Leaving mDNS multicast group on interface wlp1s0.IPv6 with address fe80::3e3c:bf3d:da61:7ea6.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Interface wlp1s0.IPv6 no longer relevant for mDNS.
Feb 26 23:35:01 eddys-pc whoopsie[1768]: [23:35:01] Cannot reach: https://daisy.ubuntu.com
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.2056] dhcp4 (wlp1s0): canceled DHCP transaction
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.2057] dhcp4 (wlp1s0): state changed bound -> done
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.2060] dhcp6 (wlp1s0): canceled DHCP transaction
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.2060] dhcp6 (wlp1s0): state changed bound -> done
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Withdrawing address record for 192.168.178.37 on wlp1s0.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Leaving mDNS multicast group on interface wlp1s0.IPv4 with address 192.168.178.37.
Feb 26 23:35:01 eddys-pc avahi-daemon[1007]: Interface wlp1s0.IPv4 no longer relevant for mDNS.
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.2119] device (wlp1s0): supplicant interface state: completed -> disconnected
Feb 26 23:35:01 eddys-pc NetworkManager[1011]: <info>  [1614378901.2124] device (wlp1s0): state change: disconnected -> unmanaged (reason 'sleeping', sys-iface-state: 'managed')
Feb 26 23:35:01 eddys-pc nm-dispatcher[4971]: run-parts: failed to stat component /etc/network/if-post-down.d/avahi-daemon: No such file or directory
Feb 26 23:35:01 eddys-pc gnome-shell[2798]: An active wireless connection, in infrastructure mode, involves no access point?
Feb 26 23:35:01 eddys-pc wpa_supplicant[1055]: nl80211: deinit ifname=p2p-dev-wlp1s0 disabled_11b_rates=0
Feb 26 23:35:01 eddys-pc wpa_supplicant[1055]: nl80211: deinit ifname=wlp1s0 disabled_11b_rates=0
Feb 26 23:35:05 eddys-pc systemd[1]: Reached target Sleep.
Feb 26 23:35:05 eddys-pc systemd[1]: Starting Suspend...
Feb 26 23:35:05 eddys-pc systemd-sleep[4999]: Suspending system...
Feb 26 23:35:05 eddys-pc kernel: [  246.154637] PM: suspend entry (deep)
Feb 26 23:35:05 eddys-pc kernel: [  246.161713] Filesystems sync: 0.007 seconds
Feb 26 23:35:21 eddys-pc kernel: [  246.286009] Freezing user space processes ... (elapsed 0.002 seconds) done.
Feb 26 23:35:21 eddys-pc kernel: [  246.288284] OOM killer disabled.
Feb 26 23:35:21 eddys-pc kernel: [  246.288285] Freezing remaining freezable tasks ... (elapsed 6.681 seconds) done.
Feb 26 23:35:21 eddys-pc kernel: [  252.969965] printk: Suspending console(s) (use no_console_suspend to debug)
Feb 26 23:35:21 eddys-pc kernel: [  252.970711] ------------[ cut here ]------------
Feb 26 23:35:21 eddys-pc kernel: [  252.970713] WARNING: CPU: 6 PID: 4999 at drivers/gpu/drm/amd/amdgpu/../display/dc/core/dc_link.c:2569 dc_link_set_backlight_level+0x92/0xf0 [amdgpu]
Feb 26 23:35:21 eddys-pc kernel: [  252.971152] Modules linked in: md4 nls_utf8 cifs fscache libdes rfcomm veth xt_nat xt_tcpudp xt_conntrack xt_MASQUERADE nf_conntrack_netlink xfrm_user xfrm_algo nft_counter xt_addrtype nft_compat nft_chain_nat nf_nat nf_conntrack nf_defrag_ipv6 nf_defrag_ipv4 nf_tables libcrc32c nfnetlink br_netfilter bridge stp llc ccm cmac algif_hash algif_skcipher af_alg overlay bnep snd_hda_codec_realtek snd_hda_codec_generic ledtrig_audio snd_hda_codec_hdmi snd_hda_intel snd_intel_dspcfg soundwire_intel soundwire_generic_allocation intel_rapl_msr soundwire_cadence snd_hda_codec snd_soc_dmic snd_acp3x_rn snd_acp3x_pdm_dma snd_hda_core iwlmvm snd_hwdep soundwire_bus nls_iso8859_1 snd_soc_core mac80211 snd_compress ac97_bus intel_rapl_common snd_pcm_dmaengine libarc4 snd_pcm iwlwifi snd_seq_midi edac_mce_amd snd_seq_midi_event serio_raw kvm_amd snd_rawmidi kvm btusb uvcvideo btrtl btbcm videobuf2_vmalloc videobuf2_memops btintel videobuf2_v4l2 videobuf2_common bluetooth efi_pstore rapl snd_seq
Feb 26 23:35:21 eddys-pc kernel: [  252.971232]  hid_multitouch snd_seq_device snd_timer wmi_bmof videodev hid_sensor_als ecdh_generic input_leds snd joydev snd_rn_pci_acp3x mc cfg80211 soundcore snd_pci_acp3x k10temp ucsi_acpi ccp ecc typec_ucsi typec ideapad_laptop sparse_keymap mac_hid hid_sensor_magn_3d hid_sensor_gyro_3d hid_sensor_accel_3d hid_sensor_trigger industrialio_triggered_buffer kfifo_buf hid_sensor_iio_common industrialio sch_fq_codel parport_pc ppdev lp parport sunrpc ip_tables x_tables autofs4 dm_crypt usbhid hid_sensor_custom amdgpu hid_sensor_hub hid_generic iommu_v2 gpu_sched i2c_algo_bit drm_ttm_helper crct10dif_pclmul crc32_pclmul ghash_clmulni_intel ttm aesni_intel drm_kms_helper crypto_simd cryptd syscopyarea glue_helper sysfillrect sysimgblt fb_sys_fops cec rc_core nvme drm ahci libahci amd_sfh xhci_pci nvme_core i2c_piix4 xhci_pci_renesas wmi video i2c_hid hid
Feb 26 23:35:21 eddys-pc kernel: [  252.971313] CPU: 6 PID: 4999 Comm: systemd-sleep Not tainted 5.11.0-051100-generic #202102142330
Feb 26 23:35:21 eddys-pc kernel: [  252.971318] Hardware name: LENOVO 82A2/LNVNB161216, BIOS DMCN27WW 05/26/2020
Feb 26 23:35:21 eddys-pc kernel: [  252.971321] RIP: 0010:dc_link_set_backlight_level+0x92/0xf0 [amdgpu]
Feb 26 23:35:21 eddys-pc kernel: [  252.971685] Code: 88 03 00 00 31 c0 48 8d 96 f0 01 00 00 48 8b 0a 48 85 c9 74 06 4c 3b 61 08 74 22 83 c0 01 48 81 c2 d0 04 00 00 83 f8 06 75 e3 <0f> 0b 45 31 ff 5b 44 89 f8 41 5c 41 5d 41 5e 41 5f 5d c3 48 98 48
Feb 26 23:35:21 eddys-pc kernel: [  252.971689] RSP: 0018:ffffa4b1c22fbbe0 EFLAGS: 00010246
Feb 26 23:35:21 eddys-pc kernel: [  252.971693] RAX: 0000000000000006 RBX: ffff8cb053eb0000 RCX: 0000000000000000
Feb 26 23:35:21 eddys-pc kernel: [  252.971695] RDX: ffff8cb0ca701ed0 RSI: ffff8cb0ca700000 RDI: 0000000000000000
Feb 26 23:35:21 eddys-pc kernel: [  252.971698] RBP: ffffa4b1c22fbc08 R08: 00000000000000ed R09: ffffa4b1c22fbbd0
Feb 26 23:35:21 eddys-pc kernel: [  252.971699] R10: 0000000000000006 R11: 0000000000000000 R12: ffff8cb053366800
Feb 26 23:35:21 eddys-pc kernel: [  252.971701] R13: 0000000000000000 R14: 000000000000eded R15: 000000000000ed01
Feb 26 23:35:21 eddys-pc kernel: [  252.971704] FS:  00007f186913d980(0000) GS:ffff8cb33f580000(0000) knlGS:0000000000000000
Feb 26 23:35:21 eddys-pc kernel: [  252.971707] CS:  0010 DS: 0000 ES: 0000 CR0: 0000000080050033
Feb 26 23:35:21 eddys-pc kernel: [  252.971710] CR2: 00005599bf91b486 CR3: 0000000105b78000 CR4: 0000000000350ee0
Feb 26 23:35:21 eddys-pc kernel: [  252.971713] Call Trace:
Feb 26 23:35:21 eddys-pc kernel: [  252.971721]  amdgpu_dm_backlight_update_status+0xbb/0xd0 [amdgpu]
Feb 26 23:35:21 eddys-pc kernel: [  252.972077]  backlight_suspend+0x71/0x80
Feb 26 23:35:21 eddys-pc kernel: [  252.972087]  ? brightness_store+0x80/0x80
Feb 26 23:35:21 eddys-pc kernel: [  252.972092]  dpm_run_callback+0x50/0x110
Feb 26 23:35:21 eddys-pc kernel: [  252.972099]  __device_suspend+0x128/0x3b0
Feb 26 23:35:21 eddys-pc kernel: [  252.972103]  dpm_suspend+0xf9/0x220
Feb 26 23:35:21 eddys-pc kernel: [  252.972107]  dpm_suspend_start+0x82/0x90
Feb 26 23:35:21 eddys-pc kernel: [  252.972111]  suspend_devices_and_enter+0x100/0x230
Feb 26 23:35:21 eddys-pc kernel: [  252.972120]  enter_state+0x131/0x184
Feb 26 23:35:21 eddys-pc kernel: [  252.972127]  pm_suspend.cold+0x22/0x6d
Feb 26 23:35:21 eddys-pc kernel: [  252.972130]  state_store+0x2b/0x60
Feb 26 23:35:21 eddys-pc kernel: [  252.972135]  kobj_attr_store+0x12/0x20
Feb 26 23:35:21 eddys-pc kernel: [  252.972141]  sysfs_kf_write+0x3f/0x50
Feb 26 23:35:21 eddys-pc kernel: [  252.972147]  kernfs_fop_write_iter+0x13b/0x1d0
Feb 26 23:35:21 eddys-pc kernel: [  252.972152]  new_sync_write+0x114/0x1a0
Feb 26 23:35:21 eddys-pc kernel: [  252.972160]  vfs_write+0x1c4/0x270
Feb 26 23:35:21 eddys-pc kernel: [  252.972164]  ksys_write+0x67/0xe0
Feb 26 23:35:21 eddys-pc kernel: [  252.972166]  __x64_sys_write+0x1a/0x20
Feb 26 23:35:21 eddys-pc kernel: [  252.972169]  do_syscall_64+0x38/0x90
Feb 26 23:35:21 eddys-pc kernel: [  252.972176]  entry_SYSCALL_64_after_hwframe+0x44/0xa9
Feb 26 23:35:21 eddys-pc kernel: [  252.972181] RIP: 0033:0x7f1869e63d57
Feb 26 23:35:21 eddys-pc kernel: [  252.972187] Code: 0d 00 f7 d8 64 89 02 48 c7 c0 ff ff ff ff eb b7 0f 1f 00 f3 0f 1e fa 64 8b 04 25 18 00 00 00 85 c0 75 10 b8 01 00 00 00 0f 05 <48> 3d 00 f0 ff ff 77 51 c3 48 83 ec 28 48 89 54 24 18 48 89 74 24
Feb 26 23:35:21 eddys-pc kernel: [  252.972189] RSP: 002b:00007fff1a1e39e8 EFLAGS: 00000246 ORIG_RAX: 0000000000000001
Feb 26 23:35:21 eddys-pc kernel: [  252.972193] RAX: ffffffffffffffda RBX: 0000000000000004 RCX: 00007f1869e63d57
Feb 26 23:35:21 eddys-pc kernel: [  252.972195] RDX: 0000000000000004 RSI: 00007fff1a1e3aa0 RDI: 0000000000000004
Feb 26 23:35:21 eddys-pc kernel: [  252.972197] RBP: 00007fff1a1e3aa0 R08: 0000000000000004 R09: 0000000000000000
Feb 26 23:35:21 eddys-pc kernel: [  252.972199] R10: 00007f1869f02040 R11: 0000000000000246 R12: 0000000000000004
Feb 26 23:35:21 eddys-pc kernel: [  252.972200] R13: 0000558e6fe472d0 R14: 0000000000000004 R15: 00007f1869f3f8c0
Feb 26 23:35:21 eddys-pc kernel: [  252.972205] ---[ end trace 6008bc85656745b5 ]---
Feb 26 23:35:21 eddys-pc kernel: [  256.995401] amdgpu: cp queue preemption time out.
Feb 26 23:35:21 eddys-pc kernel: [  257.072292] ACPI: EC: interrupt blocked
Feb 26 23:35:21 eddys-pc kernel: [  257.131560] ACPI: Preparing to enter system sleep state S3
Feb 26 23:35:21 eddys-pc kernel: [  257.132233] ACPI: EC: event blocked
Feb 26 23:35:21 eddys-pc kernel: [  257.132235] ACPI: EC: EC stopped
Feb 26 23:35:21 eddys-pc kernel: [  257.132236] PM: Saving platform NVS memory
Feb 26 23:35:21 eddys-pc kernel: [  257.132619] Disabling non-boot CPUs ...
Feb 26 23:35:21 eddys-pc kernel: [  257.133816] IRQ 50: no longer affine to CPU1
Feb 26 23:35:21 eddys-pc kernel: [  257.133836] IRQ 56: no longer affine to CPU1
Feb 26 23:35:21 eddys-pc kernel: [  257.133865] IRQ 99: no longer affine to CPU1
Feb 26 23:35:21 eddys-pc kernel: [  257.134896] smpboot: CPU 1 is now offline
Feb 26 23:35:21 eddys-pc kernel: [  257.137806] IRQ 39: no longer affine to CPU2
Feb 26 23:35:21 eddys-pc kernel: [  257.137825] IRQ 48: no longer affine to CPU2
Feb 26 23:35:21 eddys-pc kernel: [  257.137835] IRQ 52: no longer affine to CPU2
Feb 26 23:35:21 eddys-pc kernel: [  257.137842] IRQ 55: no longer affine to CPU2
Feb 26 23:35:21 eddys-pc kernel: [  257.137862] IRQ 102: no longer affine to CPU2
Feb 26 23:35:21 eddys-pc kernel: [  257.138887] smpboot: CPU 2 is now offline
Feb 26 23:35:21 eddys-pc kernel: [  257.140832] IRQ 49: no longer affine to CPU3
Feb 26 23:35:21 eddys-pc kernel: [  257.140850] IRQ 54: no longer affine to CPU3
Feb 26 23:35:21 eddys-pc kernel: [  257.141903] smpboot: CPU 3 is now offline
Feb 26 23:35:21 eddys-pc kernel: [  257.144626] smpboot: CPU 4 is now offline
Feb 26 23:35:21 eddys-pc kernel: [  257.147506] smpboot: CPU 5 is now offline
Feb 26 23:35:21 eddys-pc kernel: [  257.150319] smpboot: CPU 6 is now offline
Feb 26 23:35:21 eddys-pc kernel: [  257.153051] smpboot: CPU 7 is now offline
Feb 26 23:35:21 eddys-pc kernel: [  257.154789] ACPI: Low-level resume complete
Feb 26 23:35:21 eddys-pc kernel: [  257.154839] ACPI: EC: EC started
Feb 26 23:35:21 eddys-pc kernel: [  257.154839] PM: Restoring platform NVS memory
Feb 26 23:35:21 eddys-pc kernel: [  257.209448] LVT offset 0 assigned for vector 0x400
Feb 26 23:35:21 eddys-pc kernel: [  257.209940] Enabling non-boot CPUs ...
Feb 26 23:35:21 eddys-pc kernel: [  257.209985] x86: Booting SMP configuration:
Feb 26 23:35:21 eddys-pc kernel: [  257.209986] smpboot: Booting Node 0 Processor 1 APIC 0x1
Feb 26 23:35:21 eddys-pc kernel: [  257.210131] microcode: CPU1: patch_level=0x08600102
Feb 26 23:35:21 eddys-pc kernel: [  257.212363] ACPI: \_SB_.PLTF.C001: Found 3 idle states
Feb 26 23:35:21 eddys-pc kernel: [  257.212658] CPU1 is up
Feb 26 23:35:21 eddys-pc kernel: [  257.212681] smpboot: Booting Node 0 Processor 2 APIC 0x2
Feb 26 23:35:21 eddys-pc kernel: [  257.213111] microcode: CPU2: patch_level=0x08600102
Feb 26 23:35:21 eddys-pc kernel: [  257.215368] ACPI: \_SB_.PLTF.C002: Found 3 idle states
Feb 26 23:35:21 eddys-pc kernel: [  257.215670] CPU2 is up
Feb 26 23:35:21 eddys-pc kernel: [  257.215696] smpboot: Booting Node 0 Processor 3 APIC 0x3
Feb 26 23:35:21 eddys-pc kernel: [  257.215901] microcode: CPU3: patch_level=0x08600102
Feb 26 23:35:21 eddys-pc kernel: [  257.218088] ACPI: \_SB_.PLTF.C003: Found 3 idle states
Feb 26 23:35:21 eddys-pc kernel: [  257.218279] CPU3 is up
Feb 26 23:35:21 eddys-pc kernel: [  257.218298] smpboot: Booting Node 0 Processor 4 APIC 0x4
Feb 26 23:35:21 eddys-pc kernel: [  257.218601] microcode: CPU4: patch_level=0x08600102
Feb 26 23:35:21 eddys-pc kernel: [  257.220920] ACPI: \_SB_.PLTF.C004: Found 3 idle states
Feb 26 23:35:21 eddys-pc kernel: [  257.221166] CPU4 is up
Feb 26 23:35:21 eddys-pc kernel: [  257.221201] smpboot: Booting Node 0 Processor 5 APIC 0x5
Feb 26 23:35:21 eddys-pc kernel: [  257.221430] microcode: CPU5: patch_level=0x08600102
Feb 26 23:35:21 eddys-pc kernel: [  257.223628] ACPI: \_SB_.PLTF.C005: Found 3 idle states
Feb 26 23:35:21 eddys-pc kernel: [  257.223854] CPU5 is up
Feb 26 23:35:21 eddys-pc kernel: [  257.223875] smpboot: Booting Node 0 Processor 6 APIC 0x6
Feb 26 23:35:21 eddys-pc kernel: [  257.224107] microcode: CPU6: patch_level=0x08600102
Feb 26 23:35:21 eddys-pc kernel: [  257.226322] ACPI: \_SB_.PLTF.C006: Found 3 idle states
Feb 26 23:35:21 eddys-pc kernel: [  257.226572] CPU6 is up
Feb 26 23:35:21 eddys-pc kernel: [  257.226595] smpboot: Booting Node 0 Processor 7 APIC 0x7
Feb 26 23:35:21 eddys-pc kernel: [  257.226829] microcode: CPU7: patch_level=0x08600102
Feb 26 23:35:21 eddys-pc kernel: [  257.229051] ACPI: \_SB_.PLTF.C007: Found 3 idle states
Feb 26 23:35:21 eddys-pc kernel: [  257.229321] CPU7 is up
Feb 26 23:35:21 eddys-pc kernel: [  257.230116] ACPI: Waking up from system sleep state S3
Feb 26 23:35:21 eddys-pc kernel: [  257.231016] ACPI: EC: interrupt unblocked
Feb 26 23:35:21 eddys-pc kernel: [  257.572208] ACPI: EC: event unblocked
Feb 26 23:35:21 eddys-pc kernel: [  257.572652] pci 0000:00:00.2: can't derive routing for PCI INT A
Feb 26 23:35:21 eddys-pc kernel: [  257.572658] pci 0000:00:00.2: PCI INT A: no GSI
Feb 26 23:35:21 eddys-pc kernel: [  257.677767] [drm] Wait for DMUB auto-load failed: 3
Feb 26 23:35:21 eddys-pc kernel: [  257.677852] [drm] DMUB hardware initialized: version=0x01000000
Feb 26 23:35:21 eddys-pc kernel: [  257.772144] amdgpu 0000:03:00.0: amdgpu: ring gfx uses VM inv eng 0 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772148] amdgpu 0000:03:00.0: amdgpu: ring comp_1.0.0 uses VM inv eng 1 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772150] amdgpu 0000:03:00.0: amdgpu: ring comp_1.1.0 uses VM inv eng 4 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772151] amdgpu 0000:03:00.0: amdgpu: ring comp_1.2.0 uses VM inv eng 5 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772151] amdgpu 0000:03:00.0: amdgpu: ring comp_1.3.0 uses VM inv eng 6 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772152] amdgpu 0000:03:00.0: amdgpu: ring comp_1.0.1 uses VM inv eng 7 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772153] amdgpu 0000:03:00.0: amdgpu: ring comp_1.1.1 uses VM inv eng 8 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772153] amdgpu 0000:03:00.0: amdgpu: ring comp_1.2.1 uses VM inv eng 9 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772154] amdgpu 0000:03:00.0: amdgpu: ring comp_1.3.1 uses VM inv eng 10 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772155] amdgpu 0000:03:00.0: amdgpu: ring kiq_2.1.0 uses VM inv eng 11 on hub 0
Feb 26 23:35:21 eddys-pc kernel: [  257.772156] amdgpu 0000:03:00.0: amdgpu: ring sdma0 uses VM inv eng 0 on hub 1
Feb 26 23:35:21 eddys-pc kernel: [  257.772157] amdgpu 0000:03:00.0: amdgpu: ring vcn_dec uses VM inv eng 1 on hub 1
Feb 26 23:35:21 eddys-pc kernel: [  257.772158] amdgpu 0000:03:00.0: amdgpu: ring vcn_enc0 uses VM inv eng 4 on hub 1
Feb 26 23:35:21 eddys-pc kernel: [  257.772159] amdgpu 0000:03:00.0: amdgpu: ring vcn_enc1 uses VM inv eng 5 on hub 1
Feb 26 23:35:21 eddys-pc kernel: [  257.772160] amdgpu 0000:03:00.0: amdgpu: ring jpeg_dec uses VM inv eng 6 on hub 1
Feb 26 23:35:21 eddys-pc kernel: [  257.813145] usb 1-2: reset full-speed USB device number 2 using xhci_hcd
Feb 26 23:35:21 eddys-pc kernel: [  257.884040] ata2: SATA link down (SStatus 0 SControl 300)
Feb 26 23:35:21 eddys-pc kernel: [  257.896479] ata1: SATA link down (SStatus 0 SControl 300)
Feb 26 23:35:21 eddys-pc kernel: [  258.037860] nvme nvme0: 16/0/0 default/read/poll queues
Feb 26 23:35:21 eddys-pc kernel: [  258.803011] amdgpu 0000:03:00.0: [drm:amdgpu_ib_ring_tests [amdgpu]] *ERROR* IB test failed on gfx (-110).
Feb 26 23:35:21 eddys-pc kernel: [  258.803304] [drm:amdgpu_device_delayed_init_work_handler [amdgpu]] *ERROR* ib ring test failed (-110).
Feb 26 23:35:21 eddys-pc kernel: [  258.807287] acpi LNXPOWER:09: Turning OFF
Feb 26 23:35:21 eddys-pc kernel: [  258.807347] acpi LNXPOWER:07: Turning OFF
Feb 26 23:35:21 eddys-pc kernel: [  258.808311] OOM killer enabled.
Feb 26 23:35:21 eddys-pc kernel: [  258.808314] Restarting tasks ... done.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: The canary thread is apparently starving. Taking action.
Feb 26 23:35:21 eddys-pc systemd[1]: NetworkManager-dispatcher.service: Succeeded.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Demoting known real-time threads.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Successfully demoted thread 2835 of process 2522.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Successfully demoted thread 2834 of process 2522.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Successfully demoted thread 2744 of process 2522.
Feb 26 23:35:21 eddys-pc kernel: [  258.816934] thermal thermal_zone1: failed to read out thermal zone (-61)
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Successfully demoted thread 2522 of process 2522.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Demoted 4 threads.
Feb 26 23:35:21 eddys-pc upowerd[1353]: treating change event as add on /sys/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-2
Feb 26 23:35:21 eddys-pc upowerd[1353]: treating change event as add on /sys/devices/pci0000:00/0000:00:08.1/0000:03:00.3/usb1/1-2
Feb 26 23:35:21 eddys-pc systemd-sleep[4999]: System resumed.
Feb 26 23:35:21 eddys-pc kernel: [  258.839032] PM: suspend exit
Feb 26 23:35:21 eddys-pc systemd[1]: systemd-suspend.service: Succeeded.
Feb 26 23:35:21 eddys-pc systemd[1]: Finished Suspend.
Feb 26 23:35:21 eddys-pc systemd[1]: Stopped target Sleep.
Feb 26 23:35:21 eddys-pc systemd[1]: Reached target Suspend.
Feb 26 23:35:21 eddys-pc systemd[1]: Stopped target Suspend.
Feb 26 23:35:21 eddys-pc ModemManager[1119]: <info>  [sleep-monitor] system is resuming
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.6431] manager: sleep: wake requested (sleeping: yes  enabled: yes)
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.6432] device (wlp1s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'managed')
Feb 26 23:35:21 eddys-pc kernel: [  258.853765] [drm] Failed to add display topology, DTM TA is not initialized.
Feb 26 23:35:21 eddys-pc /usr/libexec/gdm-x-session[2617]: amdgpu: The CS has been rejected, see dmesg for more information (-16).
Feb 26 23:35:21 eddys-pc /usr/libexec/gdm-x-session[2617]: (II) AMDGPU(0): EDID vendor "AUO", prod id 26685
Feb 26 23:35:21 eddys-pc /usr/libexec/gdm-x-session[2617]: (II) AMDGPU(0): Printing DDC gathered Modelines:
Feb 26 23:35:21 eddys-pc /usr/libexec/gdm-x-session[2617]: (II) AMDGPU(0): Modeline "1920x1080"x0.0  141.00  1920 1936 1952 2104  1080 1083 1097 1116 -hsync -vsync (67.0 kHz eP)
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Supervising 0 threads of 0 processes of 0 users.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Recovering from system lockup, not allowing further RT threads.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Supervising 0 threads of 0 processes of 0 users.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Recovering from system lockup, not allowing further RT threads.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Supervising 0 threads of 0 processes of 0 users.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Recovering from system lockup, not allowing further RT threads.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Supervising 0 threads of 0 processes of 0 users.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Recovering from system lockup, not allowing further RT threads.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Supervising 0 threads of 0 processes of 0 users.
Feb 26 23:35:21 eddys-pc rtkit-daemon[1289]: Recovering from system lockup, not allowing further RT threads.
Feb 26 23:35:21 eddys-pc gnome-shell[2798]: amdgpu: The CS has been rejected, see dmesg for more information (-16).
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.8265] device (p2p-dev-wlp1s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'managed')
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.8268] manager: NetworkManager state is now CONNECTED_LOCAL
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.9085] device (wlp1s0): supplicant interface state: internal-starting -> disconnected
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.9085] device (p2p-dev-wlp1s0): state change: unavailable -> unmanaged (reason 'removed', sys-iface-state: 'removed')
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.9095] Wi-Fi P2P device controlled by interface wlp1s0 created
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.9102] manager: (p2p-dev-wlp1s0): new 802.11 Wi-Fi P2P device (/org/freedesktop/NetworkManager/Devices/16)
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.9106] device (p2p-dev-wlp1s0): state change: unmanaged -> unavailable (reason 'managed', sys-iface-state: 'external')
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.9116] device (wlp1s0): state change: unavailable -> disconnected (reason 'supplicant-available', sys-iface-state: 'managed')
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <info>  [1614378921.9124] device (p2p-dev-wlp1s0): state change: unavailable -> disconnected (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:21 eddys-pc NetworkManager[1011]: <warn>  [1614378921.9168] sup-iface[e5a867d4d138d12d,1,wlp1s0]: call-p2p-cancel: failed with P2P cancel failed
Feb 26 23:35:22 eddys-pc fwupd[3305]: 22:35:22:0115 FuEngine             csr failed to change udev device /sys/devices/pci0000:00/0000:00:08.1/0000:03:00.0/drm/card0: cannot ensure ID: FuUdevDevice:
Feb 26 23:35:22 eddys-pc fwupd[3305]: Unknown Device
Feb 26 23:35:22 eddys-pc fwupd[3305]:   Flags:                none
Feb 26 23:35:24 eddys-pc ModemManager[1119]: <info>  [base-manager] couldn't check support for device '/sys/devices/pci0000:00/0000:00:02.2/0000:01:00.0': not supported by any plugin
Feb 26 23:35:25 eddys-pc wpa_supplicant[1055]: wlp1s0: CTRL-EVENT-REGDOM-CHANGE init=DRIVER type=COUNTRY alpha2=DE
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6741] policy: auto-activating connection 'R-Quadrat24' (e443ac96-f18f-4bd9-b3ae-4f4bad7c7bc5)
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6749] device (wlp1s0): Activation: starting connection 'R-Quadrat24' (e443ac96-f18f-4bd9-b3ae-4f4bad7c7bc5)
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6756] device (wlp1s0): state change: disconnected -> prepare (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6765] manager: NetworkManager state is now CONNECTING
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6773] device (wlp1s0): state change: prepare -> config (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6779] device (wlp1s0): Activation: (wifi) access point 'R-Quadrat24' has security, but secrets are required.
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6779] device (wlp1s0): state change: config -> need-auth (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6836] device (wlp1s0): state change: need-auth -> prepare (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6842] device (wlp1s0): state change: prepare -> config (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6847] device (wlp1s0): Activation: (wifi) connection 'R-Quadrat24' has security, and secrets exist.  No new secrets needed.
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6847] Config: added 'ssid' value 'R-Quadrat24'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6847] Config: added 'scan_ssid' value '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6848] Config: added 'bgscan' value 'simple:30:-70:86400'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6848] Config: added 'key_mgmt' value 'WPA-PSK WPA-PSK-SHA256 FT-PSK'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6848] Config: added 'auth_alg' value 'OPEN'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.6848] Config: added 'psk' value '<hidden>'
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: SME: Trying to authenticate with dc:39:6f:db:0b:a9 (SSID='R-Quadrat24' freq=2472 MHz)
Feb 26 23:35:26 eddys-pc kernel: [  258.894773] [drm] Failed to add display topology, DTM TA is not initialized.
Feb 26 23:35:26 eddys-pc kernel: [  263.904845] wlp1s0: authenticate with dc:39:6f:db:0b:a9
Feb 26 23:35:26 eddys-pc kernel: [  263.907832] wlp1s0: send auth to dc:39:6f:db:0b:a9 (try 1/3)
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7008] device (wlp1s0): supplicant interface state: disconnected -> authenticating
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7008] device (p2p-dev-wlp1s0): supplicant management interface state: disconnected -> authenticating
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: Trying to associate with dc:39:6f:db:0b:a9 (SSID='R-Quadrat24' freq=2472 MHz)
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7303] device (wlp1s0): supplicant interface state: authenticating -> associating
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7304] device (p2p-dev-wlp1s0): supplicant management interface state: authenticating -> associating
Feb 26 23:35:26 eddys-pc kernel: [  263.938850] wlp1s0: authenticated
Feb 26 23:35:26 eddys-pc kernel: [  263.942805] wlp1s0: associate with dc:39:6f:db:0b:a9 (try 1/3)
Feb 26 23:35:26 eddys-pc kernel: [  263.948876] wlp1s0: RX AssocResp from dc:39:6f:db:0b:a9 (capab=0x1031 status=0 aid=5)
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: Associated with dc:39:6f:db:0b:a9
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: CTRL-EVENT-SUBNET-STATUS-UPDATE status=0
Feb 26 23:35:26 eddys-pc kernel: [  263.955048] wlp1s0: associated
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7515] device (wlp1s0): supplicant interface state: associating -> 4way_handshake
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7516] device (p2p-dev-wlp1s0): supplicant management interface state: associating -> 4way_handshake
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: WPA: Key negotiation completed with dc:39:6f:db:0b:a9 [PTK=CCMP GTK=CCMP]
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: CTRL-EVENT-CONNECTED - Connection to dc:39:6f:db:0b:a9 completed [id=0 id_str=]
Feb 26 23:35:26 eddys-pc kernel: [  263.980821] IPv6: ADDRCONF(NETDEV_CHANGE): wlp1s0: link becomes ready
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: CTRL-EVENT-SIGNAL-CHANGE above=0 signal=-55 noise=9999 txrate=52000
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7772] device (wlp1s0): supplicant interface state: 4way_handshake -> completed
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7773] device (wlp1s0): Activation: (wifi) Stage 2 of 5 (Device Configure) successful. Connected to wireless network "R-Quadrat24"
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7773] device (p2p-dev-wlp1s0): supplicant management interface state: 4way_handshake -> completed
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7776] device (wlp1s0): state change: config -> ip-config (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.7782] dhcp4 (wlp1s0): activation: beginning transaction (timeout in 45 seconds)
Feb 26 23:35:26 eddys-pc avahi-daemon[1007]: Joining mDNS multicast group on interface wlp1s0.IPv6 with address fe80::3e3c:bf3d:da61:7ea6.
Feb 26 23:35:26 eddys-pc avahi-daemon[1007]: New relevant interface wlp1s0.IPv6 for mDNS.
Feb 26 23:35:26 eddys-pc avahi-daemon[1007]: Registering new address record for fe80::3e3c:bf3d:da61:7ea6 on wlp1s0.*.
Feb 26 23:35:26 eddys-pc kernel: [  264.042155] wlp1s0: Limiting TX power to 20 (20 - 0) dBm as advertised by dc:39:6f:db:0b:a9
Feb 26 23:35:26 eddys-pc wpa_supplicant[1055]: wlp1s0: CTRL-EVENT-SIGNAL-CHANGE above=1 signal=-54 noise=9999 txrate=52000
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8483] dhcp4 (wlp1s0): option dhcp_lease_time      => '864000'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8484] dhcp4 (wlp1s0): option domain_name          => 'fritz.box'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8484] dhcp4 (wlp1s0): option domain_name_servers  => '192.168.178.1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8484] dhcp4 (wlp1s0): option expiry               => '1615242926'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8484] dhcp4 (wlp1s0): option ip_address           => '192.168.178.37'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8484] dhcp4 (wlp1s0): option next_server          => '192.168.178.1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8484] dhcp4 (wlp1s0): option ntp_servers          => '192.168.178.1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8484] dhcp4 (wlp1s0): option requested_broadcast_address => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_domain_name => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_domain_name_servers => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_domain_search => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_host_name  => '1'
Feb 26 23:35:26 eddys-pc avahi-daemon[1007]: Joining mDNS multicast group on interface wlp1s0.IPv4 with address 192.168.178.37.
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_interface_mtu => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_ms_classless_static_routes => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_nis_domain => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_nis_servers => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8485] dhcp4 (wlp1s0): option requested_ntp_servers => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option requested_rfc3442_classless_static_routes => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option requested_root_path  => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option requested_routers    => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option requested_static_routes => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option requested_subnet_mask => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option requested_time_offset => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option requested_wpad       => '1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option routers              => '192.168.178.1'
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8486] dhcp4 (wlp1s0): option subnet_mask          => '255.255.255.0'
Feb 26 23:35:26 eddys-pc avahi-daemon[1007]: New relevant interface wlp1s0.IPv4 for mDNS.
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8487] dhcp4 (wlp1s0): state changed unknown -> bound
Feb 26 23:35:26 eddys-pc avahi-daemon[1007]: Registering new address record for 192.168.178.37 on wlp1s0.IPv4.
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8510] device (wlp1s0): state change: ip-config -> ip-check (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc dbus-daemon[1010]: [system] Activating via systemd: service name='org.freedesktop.nm_dispatcher' unit='dbus-org.freedesktop.nm-dispatcher.service' requested by ':1.13' (uid=0 pid=1011 comm="/usr/sbin/NetworkManager --no-daemon ")
Feb 26 23:35:26 eddys-pc systemd[1]: Starting Network Manager Script Dispatcher Service...
Feb 26 23:35:26 eddys-pc dbus-daemon[1010]: [system] Successfully activated service 'org.freedesktop.nm_dispatcher'
Feb 26 23:35:26 eddys-pc systemd[1]: Started Network Manager Script Dispatcher Service.
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8618] device (wlp1s0): state change: ip-check -> secondaries (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8621] device (wlp1s0): state change: secondaries -> activated (reason 'none', sys-iface-state: 'managed')
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8626] manager: NetworkManager state is now CONNECTED_LOCAL
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8647] manager: NetworkManager state is now CONNECTED_SITE
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8648] policy: set 'R-Quadrat24' (wlp1s0) as default for IPv4 routing and DNS
Feb 26 23:35:26 eddys-pc NetworkManager[1011]: <info>  [1614378926.8667] device (wlp1s0): Activation: successful, device activated.
Feb 26 23:35:27 eddys-pc NetworkManager[1011]: <info>  [1614378927.1766] manager: NetworkManager state is now CONNECTED_GLOBAL
Feb 26 23:35:29 eddys-pc NetworkManager[1011]: <info>  [1614378929.0913] dhcp6 (wlp1s0): activation: beginning transaction (timeout in 45 seconds)
Feb 26 23:35:29 eddys-pc avahi-daemon[1007]: Leaving mDNS multicast group on interface wlp1s0.IPv6 with address fe80::3e3c:bf3d:da61:7ea6.
Feb 26 23:35:29 eddys-pc avahi-daemon[1007]: Joining mDNS multicast group on interface wlp1s0.IPv6 with address 2003:e9:2714:2000:f4e7:cc38:c973:8063.
Feb 26 23:35:29 eddys-pc avahi-daemon[1007]: Registering new address record for 2003:e9:2714:2000:f4e7:cc38:c973:8063 on wlp1s0.*.
Feb 26 23:35:29 eddys-pc avahi-daemon[1007]: Withdrawing address record for fe80::3e3c:bf3d:da61:7ea6 on wlp1s0.
Feb 26 23:35:29 eddys-pc NetworkManager[1011]: <info>  [1614378929.0928] policy: set 'R-Quadrat24' (wlp1s0) as default for IPv6 routing and DNS
Feb 26 23:35:29 eddys-pc NetworkManager[1011]: <info>  [1614378929.0973] dhcp6 (wlp1s0): option dhcp6_name_servers   => 'fd00::de39:6fff:fef9:f8df'
Feb 26 23:35:29 eddys-pc NetworkManager[1011]: <info>  [1614378929.0973] dhcp6 (wlp1s0): state changed unknown -> bound
Feb 26 23:35:30 eddys-pc avahi-daemon[1007]: Registering new address record for 2003:e9:2714:2000:70d1:a6e0:3188:80b3 on wlp1s0.*.
Feb 26 23:35:30 eddys-pc kernel: [  264.179468] [drm:amdgpu_dm_commit_planes.constprop.0 [amdgpu]] *ERROR* Waiting for fences timed out!
Feb 26 23:35:30 eddys-pc kernel: [  268.018964] [drm:amdgpu_job_timedout [amdgpu]] *ERROR* ring sdma0 timeout, signaled seq=1336, emitted seq=1339
Feb 26 23:35:30 eddys-pc kernel: [  268.019303] [drm:amdgpu_job_timedout [amdgpu]] *ERROR* Process information: process  pid 0 thread  pid 0
Feb 26 23:35:30 eddys-pc kernel: [  268.019601] amdgpu 0000:03:00.0: amdgpu: GPU reset begin!
Feb 26 23:35:31 eddys-pc kernel: [  268.272406] amdgpu 0000:03:00.0: [drm:amdgpu_ring_test_helper [amdgpu]] *ERROR* ring kiq_2.1.0 test failed (-110)
Feb 26 23:35:31 eddys-pc kernel: [  268.282847] [drm] free PSP TMR buffer
Feb 26 23:35:31 eddys-pc kernel: [  268.315955] amdgpu 0000:03:00.0: amdgpu: MODE2 reset
Feb 26 23:35:31 eddys-pc kernel: [  268.316111] amdgpu 0000:03:00.0: amdgpu: GPU reset succeeded, trying to resume
Feb 26 23:35:31 eddys-pc kernel: [  268.316308] [drm] PCIE GART of 1024M enabled (table at 0x000000F400900000).
Feb 26 23:35:31 eddys-pc kernel: [  268.316556] [drm] PSP is resuming...
Feb 26 23:35:31 eddys-pc kernel: [  268.336447] [drm] reserve 0x400000 from 0xf41f800000 for PSP TMR
Feb 26 23:35:31 eddys-pc kernel: [  268.514983] amdgpu 0000:03:00.0: amdgpu: SMU is resuming...
Feb 26 23:35:31 eddys-pc kernel: [  268.516548] amdgpu 0000:03:00.0: amdgpu: SMU is resumed successfully!
Feb 26 23:35:31 eddys-pc kernel: [  268.518490] [drm] kiq ring mec 2 pipe 1 q 0
Feb 26 23:35:31 eddys-pc kernel: [  268.532688] [drm] DMUB hardware initialized: version=0x01000000
Feb 26 23:35:31 eddys-pc kernel: [  268.689082] [drm] Failed to add display topology, DTM TA is not initialized.
Feb 26 23:35:31 eddys-pc kernel: [  268.693831] [drm] Failed to add display topology, DTM TA is not initialized.
Feb 26 23:35:31 eddys-pc kernel: [  268.740132] [drm] VCN decode and encode initialized successfully(under DPG Mode).
Feb 26 23:35:31 eddys-pc kernel: [  268.740763] [drm] JPEG decode initialized successfully.
Feb 26 23:35:31 eddys-pc kernel: [  268.740770] amdgpu 0000:03:00.0: amdgpu: ring gfx uses VM inv eng 0 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740776] amdgpu 0000:03:00.0: amdgpu: ring comp_1.0.0 uses VM inv eng 1 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740779] amdgpu 0000:03:00.0: amdgpu: ring comp_1.1.0 uses VM inv eng 4 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740781] amdgpu 0000:03:00.0: amdgpu: ring comp_1.2.0 uses VM inv eng 5 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740784] amdgpu 0000:03:00.0: amdgpu: ring comp_1.3.0 uses VM inv eng 6 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740786] amdgpu 0000:03:00.0: amdgpu: ring comp_1.0.1 uses VM inv eng 7 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740788] amdgpu 0000:03:00.0: amdgpu: ring comp_1.1.1 uses VM inv eng 8 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740791] amdgpu 0000:03:00.0: amdgpu: ring comp_1.2.1 uses VM inv eng 9 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740793] amdgpu 0000:03:00.0: amdgpu: ring comp_1.3.1 uses VM inv eng 10 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740796] amdgpu 0000:03:00.0: amdgpu: ring kiq_2.1.0 uses VM inv eng 11 on hub 0
Feb 26 23:35:31 eddys-pc kernel: [  268.740799] amdgpu 0000:03:00.0: amdgpu: ring sdma0 uses VM inv eng 0 on hub 1
Feb 26 23:35:31 eddys-pc kernel: [  268.740801] amdgpu 0000:03:00.0: amdgpu: ring vcn_dec uses VM inv eng 1 on hub 1
Feb 26 23:35:31 eddys-pc kernel: [  268.740804] amdgpu 0000:03:00.0: amdgpu: ring vcn_enc0 uses VM inv eng 4 on hub 1
Feb 26 23:35:31 eddys-pc kernel: [  268.740806] amdgpu 0000:03:00.0: amdgpu: ring vcn_enc1 uses VM inv eng 5 on hub 1
Feb 26 23:35:31 eddys-pc kernel: [  268.740808] amdgpu 0000:03:00.0: amdgpu: ring jpeg_dec uses VM inv eng 6 on hub 1
Feb 26 23:35:31 eddys-pc gnome-shell[2798]: Removing a network device that was not added
Feb 26 23:35:31 eddys-pc kernel: [  268.747383] amdgpu 0000:03:00.0: amdgpu: recover vram bo from shadow start
Feb 26 23:35:31 eddys-pc kernel: [  268.747391] amdgpu 0000:03:00.0: amdgpu: recover vram bo from shadow done
Feb 26 23:35:31 eddys-pc kernel: [  268.747466] amdgpu 0000:03:00.0: amdgpu: GPU reset(1) succeeded!
Feb 26 23:35:31 eddys-pc whoopsie[1768]: [23:35:31] The default IPv4 route is: /org/freedesktop/NetworkManager/ActiveConnection/12
Feb 26 23:35:31 eddys-pc whoopsie[1768]: [23:35:31] Not a paid data plan: /org/freedesktop/NetworkManager/ActiveConnection/12
Feb 26 23:35:31 eddys-pc whoopsie[1768]: [23:35:31] Found usable connection: /org/freedesktop/NetworkManager/ActiveConnection/12
Feb 26 23:35:31 eddys-pc whoopsie[1768]: [23:35:31] online
Feb 26 23:35:31 eddys-pc whoopsie[1768]: [23:35:31] Cannot reach: https://daisy.ubuntu.com
Feb 26 23:35:31 eddys-pc whoopsie[1768]: [23:35:31] offline
Feb 26 23:35:32 eddys-pc whoopsie[1768]: [23:35:32] online
Feb 26 23:35:37 eddys-pc systemd-resolved[968]: Using degraded feature set UDP instead of UDP+EDNS0 for DNS server 192.168.178.1.
Feb 26 23:35:38 eddys-pc systemd[1]: NetworkManager-dispatcher.service: Succeeded.
Feb 26 23:36:32 eddys-pc systemd-modules-load[584]: Inserted module 'lp'
Feb 26 23:36:32 eddys-pc systemd-modules-load[584]: Inserted module 'ppdev'
Feb 26 23:36:32 eddys-pc kernel: [    0.000000] Linux version 5.11.0-051100-generic (kernel@kathleen) (gcc (Ubuntu 10.2.0-13ubuntu1) 10.2.0, GNU ld (GNU Binutils for Ubuntu) 2.35.1) #202102142330 SMP Sun Feb 14 23:33:21 UTC 2021
`

**Comment by jrandiny** [comment]
Sorry for the late response, can you specify your Ubuntu version and the way you install Kernel 5.11

**Comment by eddykaya** [comment]
Hey @jrandiny,

no worries. I use Ubuntu 20.10 and installed the kernel 5.11.0 and 5.11.2 via `dpkg -i *.deb`. I downloaded the necessary packages from (here)[https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.11/] and (here)[https://kernel.ubuntu.com/~kernel-ppa/mainline/v5.11.2/]
(headers, headers-generci, image and modules)

**Comment by Cubxity** [comment]
I also have this issue on Arch Linux with Kernel version 5.11.2 (linux-zen)

**Comment by jrandiny** [comment]
Hi, it seems that I can't replicate the problem on my system (Ubuntu 20.04, kernel 5.11.2 installed via https://github.com/pimlie/ubuntu-mainline-kernel.sh)

**Comment by jrandiny** [comment]
There seems to be some kind of regression on kernel 5.11. I have tested 5.11.2 and 5.11.3 and encountered the same problem. Until a fix is found, you can try with kernel 5.10.20 or stick with the default 5.8 kernel (I am currently daily driving kernel 5.8 and haven't found any problem)

**Comment by elrandar** [comment]
Hibernation also not working properly on this kernel version.
https://bbs.archlinux.org/viewtopic.php?id=264044

**Comment by moll** [comment]
I can confirm the suspend problem on Fedora 33 running on Yoga Slim 7 14ARE05 with Linux v5.11.

**Comment by sarvasana** [comment]
Fixed on 5.12.

**Comment by devillemereuil** [comment]
I cannot confirm the issue being fixed with 5.12 on openSUSE Tumbleweed.

@sarvasana @eddykaya Did you guys change anything from this repo's README to make this work?

**Comment by rom1v** [comment]
Suspend does not work anymore on Debian sid too:
```
$ uname -a
Linux yoga 5.10.0-6-amd64 #1 SMP Debian 5.10.28-1 (2021-04-09) x86_64 GNU/Linux
```

I could not make these steps work anymore: https://github.com/jrandiny/yoga-slim7-ubuntu#advertise-s3

`acpidump -b` now produces `dsdt1.dat` and `dsdt2.dat` (among other files) instead of a single `dsdt.dat`, and I can't generate a non-empty `.dsl` file anymore.

`man iasl` says:
```
       -da [<file1>,<file2>]
              Disassemble multiple tables from single namespace
```

```bash
$ iasl -e *.dat -da dsdt1.dat,dsdt2.dat

Intel ACPI Component Architecture
ASL+ Optimizing Compiler/Disassembler version 20200925
Copyright (c) 2000 - 2020 Intel Corporation

Error    6092 - Could not open file "dsdt1.dat,dsdt2.dat" (Source Input) - No such file or directory
```
:thinking:

**Comment by der-martin85** [comment]
I'm on Ubuntu 21.04 mit Kernel 5.11 and its not working. I tried to install the old Kernel 5.8 again and also the Mainline Kernels 5.12.9 and 5.13-rc4 but had no luck making it work.

**Comment by der-martin85** [comment]
> I'm on Ubuntu 21.04 with Kernel 5.11 and its not working. I tried to install the old Kernel 5.8 again and also the Mainline Kernels 5.12.9 and 5.13-rc4 but had no luck making it work.

I got it working again with Kernel 5.10 as suggested in this comment:
https://github.com/jrandiny/yoga-slim7-ubuntu/issues/31#issuecomment-855404396

**Comment by cies** [comment]
I had luck with a 5.13.9 kernel on  Ubuntu 21.04. See issue #48 .

**Comment by devillemereuil** [comment]
An update on openSUSE Tumbleweed: suspend is now working with kernel 5.13.8!

**Comment by jrandiny** [comment]
Hm it seems that a lot of people reported that kernel 5.13 works, I will test this. For anyone on kernel 5.11 I think the best option is to use the bios unlock method or rollback to previous version of the kernel (5.8 or 5.10). For other problem please open a new issues. Thanks :)

**Comment by acid24** [comment]
Suspend seems to be working again on kernel 5.11.0.37 (Ubuntu 20.04.3 on Yoga Slim 7 4700U)
